src/apps/testapps/testVertex.c

```diff
@@ -85,4 +85,35 @@ SUITE(Vertex) {
         t_assert(directionForVertexNum(pentagon, 5) == INVALID_DIGIT,
                  "invalid pent vertex should return invalid direction");
     }
+
+    TEST(isValidVertex_hex) {
```

Comment:
Please add a test for a specific valid vertex, to validate the right mode number is being used.

Reply:
Will do.